Makefile

```diff
@@ -27,9 +27,11 @@ proto:
 	done
 
 test: testdeps
+	go test -i google.golang.org/grpc/...
```

Comment:
What I'm saying is that this something specifically incorrect about your setup, and it will not affect most users because it only interacts with GOPATH and GOPATH should not be owned by root.

I'm happy to help you debug your setup - come join our gitter channel https://gitter.im/cockroachdb/cockroach.